src/buildHTML.js

```diff
@@ -392,12 +439,17 @@ groupTypes.genfrac = function(group, options, prev) {
 
     var nstyle = style.fracNum();
     var dstyle = style.fracDen();
+    var newOptions;
 
-    var numer = buildGroup(group.value.numer, options.withStyle(nstyle));
-    var numerreset = makeSpan([style.reset(), nstyle.cls()], [numer]);
+    newOptions = options.withStyle(nstyle);
+    var numer = buildGroup(group.value.numer, newOptions);
+    var numerreset = makeSpan([style.reset(), nstyle.cls()],
+        [numer], newOptions);
```

Comment:
Yeah no look let me push back on this. There are `makeSpan` calls in `delimiter.js`, etc., that as of this commit have no `options` to pass. How about I put a TODO to improve API consistency.